apps/frontend/components/pages/Admin/Admin.tsx

```diff
@@ -0,0 +1,22 @@
+import AddIcon from '@mui/icons-material/Add';
+import { Box, Button, Typography } from '@mui/material';
+
+export const Admin = (): JSX.Element => {
+  return (
+    <Box display="flex" flexDirection="row-reverse" width="100%" pr={2}>
+      <Button
+        variant="contained"
+        size="large"
+        sx={{
+          borderRadius: 100,
+          boxShadow: 6,
+        }}
+      >
+        <Typography variant="h6" padding={1}>
+          Add User
```

Comment:
Let's strart using react intl

Reply:
There are a couple diff approaches with that, so am deferring the decision til we do the actual internationalization ticket. Will make note of texts that need change in teh ticket.